interface/login/login.php

```diff
@@ -22,6 +23,8 @@
  * @author  Julia Longtin <julialongtin@diasp.org>
  * @author  cfapress
  * @author  markleeds
+ * @author  Robert Down <robertdown@live.com
+ * @license https://www.gnu.org/licenses/gpl.html GNU GPL 3
```

Comment:
Don't include a license tag. We decided to not do this to avoid making it more difficult to migrate to next version of GPL license when needed (best to have it as it is in the license wordage above).